workflow-templates/im-deploy-on-prem-database.yml

```diff
@@ -177,6 +180,32 @@ jobs:
           DB_SERVER_NAME@stage demo: ''
           DB_SERVER_NAME@prod: ''
 
+      - name: AZ Login
+        uses: azure/login@v1
+        with:
+          # TODO: These the Service Principal these credentials are for needs access to Vault. See this git-er-done article for more information:
+          # https://github.com/im-practices/git-er-done/blob/main/actions/hashicorp-vault-integration.md#setup
+          creds: ${{ secrets.AZURE_LOGIN_CREDENTIALS }}
```

Comment:
If these are the same JSON formatted creds that we use in our tf, should we call it AZURE_CREDENTIALS?  That's how the secret is referred to everywhere else.